crates/core/tedge/src/cli/connect/command.rs

```diff
@@ -14,6 +14,7 @@ const WAIT_FOR_CHECK_SECONDS: u64 = 2;
 const C8Y_CONFIG_FILENAME: &str = "c8y-bridge.conf";
 const AZURE_CONFIG_FILENAME: &str = "az-bridge.conf";
 pub(crate) const RESPONSE_TIMEOUT: Duration = Duration::from_secs(10);
+pub(crate) const CONNECTION_TIMEOUT: u64 = 60;
```

Comment:
Use a `Duration` here and call [`Duration::as_secs_f64`](https://doc.rust-lang.org/std/time/struct.Duration.html#method.as_secs_f64) when setting the MQTT client option.

Reply:
The signature is `pub fn set_connection_timeout(&mut self, timeout: u64)` takes u64. So, I had to change it to `CONNECTION_TIMEOUT.as_secs()`